props/saber_fett263_buttons.h

```diff
@@ -2053,6 +2053,18 @@ SaberFett263Buttons() : PropBase() {}
     }
   }
 
+  void PlayEffectTrack() {
+    if (!track_player_->isPlaying()) {
+      num_tracks_ = RunCommandAndGetSingleLine("list_current_tracks", nullptr, 0, 0, 0);
```

Comment:
You should probably cache this so you don't need to do it while the saber is on.

Reply:
Moved to chdir() override {}